store/postgres/src/subgraph_store.rs

```diff
@@ -1320,6 +1320,14 @@ impl SubgraphStoreTrait for SubgraphStore {
         self.mirror.subgraph_exists(name)
     }
 
+    fn subgraph_features(
+        &self,
+        deployment: &DeploymentHash,
+    ) -> Result<Option<DeploymentFeatures>, StoreError> {
+        self.primary_conn()?
+            .get_subgraph_features(deployment.to_string())
```

Comment:
This should be made async. We should never try to get a DB connection from async code, as that is a blocking operation, so `with_conn` must be used.